main.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import FastAPI, Query
+import api
+
+app = FastAPI()
+
+
+@app.get('/search')
+def search_flights(from_city: str = Query(...), to_city: str = Query(...), date: str = Query(...)): 
```

Comment:
What does `Query(...)` mean?

Reply:
In FastAPI, `Query(...)` is used to declare additional validation and metadata for HTTP query parameters. The ellipsis `...` is used to make the parameter required. So, `from_city: str = Query(...)` means that the `from_city` query parameter is required and should be of type `str`. If the parameter is not provided in the request, FastAPI will return a 422 Unprocessable Entity response detailing the issue.